src/blockchain/chain.rs

```diff
@@ -594,8 +595,9 @@ impl Chain {
             .combine(&public_key_bill_seller.inner)
             .unwrap();
         let pub_key_bill = bitcoin::PublicKey::new(public_key_bill);
-
-        bitcoin::Address::p2pkh(pub_key_bill, USEDNET).to_string()
+        let conf = Config::try_parse();
```

Comment:
We have to only parse the config on startup, otherwise we might get into an inconsistent state (e.g. if the file changes).

So the config created in `main.rs`.

One idea would be to use something like once_cell::Lazy, or lazy_static, to initialize the config as a constant and then we can use it from anywhere without bothering about this.